verify_ehc.py

```diff
@@ -817,13 +820,22 @@ def download_nl_certs(token: Optional[str] = None) -> CertList:
     response.raise_for_status()
     certs_json = json.loads(response.content)
 
-    payload   = b64decode(certs_json['payload'])
+    payload   = certs_json['payload']
 
     # Signature is a CMS (rfc5652) detached signature of the payload.
     # The certificate chain in this pkcs#7 signature rolls up to the
     # rootkey of the Kingdom of the Netherlands (https://www.pkioverheid.nl)
     #
-    signature = b64decode(certs_json['signature'])
+    signature = certs_json['signature']
+
+    if cacert:
+        if pkcs7_detached.verify_detached_signature(payload, signature, cacert) != True:
+            print_err("Signature on the trustfile of the Netherlands did not verify against the countries National PKI root")
+            exit(1)
```

Comment:
Don't call `exit(1)`, `raise` an exception.

Reply:
Also are you sure this cannot be done with the `cryptography` library instead of yet another crypto library?